subworkflows/nf-core/abundance_differential_filter/main.nf

```diff
@@ -138,7 +138,7 @@ workflow ABUNDANCE_DIFFERENTIAL_FILTER {
 
     // DREAM only runs with formula
     dream_inputs = inputs.contrasts_for_diff_with_formula
-        .filter { meta, variable, reference, target, formula ->
+        .filter { meta, _variable, _reference, _target, formula, _comparison ->
```

Comment:
The module supports both, doesn't it? 
(You wouldn't be able to specify mixed effects, but in that case DREAM just behaves like limma)

Reply:
Yes, the module can support both. However for this version of the subworkflow, I would prefer to focus only on supporting the formula.